packages/bot-web-ui/src/utils/__tests__/download.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { LogTypes } from '@deriv/bot-skeleton';
+import { localize } from '@deriv/translations';
+import { getCurrentDateTimeLocale, getSuccessJournalMessage } from 'Utils/download';
+
+jest.mock('@deriv/translations', () => ({
+    localize: jest.fn((message, values) => {
+        let result = message;
+        if (values) {
+            Object.keys(values).forEach(key => {
+                result = result.replace(`{{${key}}}`, values[key]);
+            });
+        }
+        return result;
+    }),
+}));
+jest.mock('@deriv/bot-skeleton', () => ({
+    LogTypes: {
```

Comment:
we have already exported `LogTypes` from skeleton can we use it from here 
import { LogTypes } from '@/external/bot-skeleton';

insted of defining the types something  like this 

import { LogTypes } from '@deriv/bot-skeleton';

jest.mock('@deriv/bot-skeleton', () => ({
    LogTypes: LogTypes,
}));

Reply:
hey @rupato-deriv 
i faced a error in download due to this 
, actually its running the dbot.js. 
so thats why i mocked those values.

Reply:
ok will check this ✅